### PR TITLE
Add user preference combobox for BSQ block explorer with random default

### PR DESCRIPTION
**Pull request by wiz**
Screen Shot 2019-12-16 at 0 21 25

Implements https://github.com/bisq-network/proposals/issues/143

Needs review and testing

### CODE REVIEW

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 1]
```diff
@@ -107,12 +108,18 @@
...
-    private static final BlockChainExplorer BSQ_BETA_NET_EXPLORER = new BlockChainExplorer("BSQ", "http://explorer.betanet.bisq.network/tx.html?tx=",
-            "http://explorer.betanet.bisq.network/Address.html?addr=");
-    private static final BlockChainExplorer BSQ_TEST_NET_EXPLORER = new BlockChainExplorer("BSQ", "http://explorer.testnet.bisq.network/tx.html?tx=",
-            "http://explorer.testnet.bisq.network/Address.html?addr=");
+    public static final ArrayList<BlockChainExplorer> BSQ_MAIN_NET_EXPLORERS = new ArrayList<>(Arrays.asList(
+            new BlockChainExplorer("bsq.wiz.biz (@wiz)", "https://bsq.wiz.biz/tx.html?tx=", "https://bsq.wiz.biz/Address.html?addr="),
+            new BlockChainExplorer("explorer.sqrrm.net (@sqrrm)", "https://explorer.sqrrm.net/tx.html?tx=", "https://explorer.sqrrm.net/Address.html?addr=")
+    ));
+
+    private static final ArrayList<BlockChainExplorer> BSQ_BETA_NET_EXPLORERS = new ArrayList<>(Collections.singletonList(
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/user/Preferences.java]: Do we still run betanet exploreres? I think its not worth the effort if so.

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 2]
```diff
@@ -107,12 +108,18 @@
...
+            new BlockChainExplorer("BSQ", "http://explorer.betanet.bisq.network/tx.html?tx=", "http://explorer.betanet.bisq.network/Address.html?addr=")
+    ));
+
+    private static final ArrayList<BlockChainExplorer> BSQ_TEST_NET_EXPLORERS = new ArrayList<>(Collections.singletonList(
+            new BlockChainExplorer("BSQ", "http://explorer.testnet.bisq.network/tx.html?tx=", "http://explorer.testnet.bisq.network/Address.html?addr=")
```
Comment (chimp1984) [inline core/src/main/java/bisq/core/user/Preferences.java]: Same with testnet. I doubt there is a testnet explorer running and testnet is pretty un-usable anyway for app testing.

**Review by chimp1984 — DISMISSED** [review]
utACK

IMO betanet and testnet can be removed...

File: core/src/main/java/bisq/core/user/Preferences.java (hunk 1 above)
Comment (ripcurlx) [reply core/src/main/java/bisq/core/user/Preferences.java]: Yes, they are both not reachable ATM. I think we can remove them from here.

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - please see the comments from @chimp1984 and myself.

File: core/src/main/java/bisq/core/user/Preferences.java (hunk 1 above)
Comment (sqrrm) [reply core/src/main/java/bisq/core/user/Preferences.java]: I haven't run any betanet explorers for almost a year. Doubt anyone else has either.

**Review by sqrrm — DISMISSED** [review]
utACK

**Review by ripcurlx — APPROVED** [review]
ACK

Tested it on Mainnet and everything works as expected for @wiz explorer. @sqrrm explorer should be available for our v1.2.5 release

**Pull request merged**